lib/screen/weather_screen.dart

```diff
@@ -16,8 +16,44 @@ class _WeatherScreenState extends State<WeatherScreen> {
   String _weatherImage = '';
 
   void _reloadWeather() {
-    _weatherImage = _weather.fetchSimpleWeather();
-    setState(() {});
+    try {
+      _weatherImage = _weather.fetchThrowsWeather('tokyo');
+      setState(() {});
+    } on YumemiWeatherError catch (error) {
+      final errorMessage = _convertYumemiWeatherError(error);
+      _openErrorDialog(errorMessage);
+    }
+  }
+
+  String _convertYumemiWeatherError(YumemiWeatherError error) {
+    var errorMessage = '';
```

Comment:
[nits]
見逃していましたが、このようにしたほうがより適切そうです！

```suggestion
    final String errorMessage;
```

Reply:
見逃していました。。ご指摘ありがとうございます。